test-harness/specs/security-scheme-validation-AC-24.oas2.txt

```diff
@@ -0,0 +1,34 @@
+====test====
+Providing two required API KEYS grants access to a resource.
+====spec====
+swagger: '2.0'
+paths:
+  /todos:
+    get:
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          examples:
+            application/json: test
+securityDefinitions:
+  ApiKeyOne:
+    type: apiKey
+    in: header
+    name: X-API-Key
+  ApiKeyTwo:
+    type: apiKey
+    in: query
+    name: apiKey
+security:
+  - ApiKeyOne: []
+    ApiKeyTwo: []
+====server====
+mock -p 4010
+====command====
+curl -i http://localhost:4010/todos?apiKey=abc123 -H "X-API-Key: xyz987"
+====expect====
+HTTP/1.1 200 OK
+
+"test"
```

Comment:
we dont always need to have a json string output at the end, a empty 200 OK should be fine. If you remove the JSON and the example it'll flop our a text/plan 200 OK right?

Reply:
@Phil, yes, but I wanted to keep `oas2` test and `oas3` test as close as possible. And since doing what you described for `oas3` results in `406` response, I went with an example JSON response for the both tests. Hope it makes sense

Reply:
I don't follow. I'm saying this test is about security scheme validation, not about JSON examples, so im not sure why we've added JSON examples?

Reply:
Hm, so you're saying that if the response is the default `text/plain`, then all of a sudden, it only is about security scheme validation and not about `text/plain`? How is it different that having `text/plain` default response is not testing this part of Prism and only testing security scheme validation? But, having said that... as I said, I wanted to have `test-harness/specs/security-scheme-validation-AC-24.oas2.txt` and `test-harness/specs/security-scheme-validation-AC-24.oas3.txt` (and all other `oas2`/`oas3` pairs) as close as possible.

So, how do I make `oas3` example returning `text/plain` default answer? If I adjust `oas3` file the was I adjust it for `oas2`, I get `406`.

Reply:
I'm saying we dont need to add extra lines to the test in order to create a JSON example which we dont need. By removing irrelevant things we slim the test down to only relevant things... I'm not sure I understand your concerns here, or what you're getting at. I was just asking if we could remove irrelevant things.

Reply:
@philsturgeon, here's the change https://github.com/stoplightio/prism/pull/514/commits/dff5f9ee5b05afccde19e5fbae64b0e23cd8eda6